web/src/settings_notifications.ts

```diff
@@ -371,8 +371,6 @@ export function set_up(settings_panel: SettingsPanel): void {
         settings_object.automatically_unmute_topics_in_muted_streams_policy,
     );
 
-    update_desktop_notification_banner();
```

Comment:
Why is this being removed?

Reply:
OK but that function is also called in the main personal notifications panel. Are you breaking that?

It's really quite frustrating to see changes like this where it doesn't seem that either the PR description or the code changes engage with the fact that this banner appears in two places, and the spec only requests we only remove one.